app/src/main/res/values/strings.xml

```diff
@@ -1,14 +1,17 @@
 <resources>
     <string name="app_name">WNews</string>
-    <string name="title_activity_login">LoginActivity</string>
-    <string name="email_user">Email</string>
-    <string name="password_user">Password</string>
-    <string name="log_in">Log In</string>
-    <string name="sing_up">Sing Up</string>
-    <string name="terms">By joining you agree to our Terms and Conditions</string>
-    <string name="hello_blank_fragment">Hello blank fragment</string>
+
+    <!-- Login -->
     <string name="email_error">It is not an email</string>
     <string name="mail_missing">Mail is missing</string>
     <string name="password_missing">Password is missing</string>
     <string name="fail_login">Invalid email or password</string>
+    <string name="terms">By joining you agree to our Terms and Conditions</string>
```

Comment:
Use a prefix for strings related to the same functionality:

For example for the Login section, I added below the `login_` prefix to each ID.

```xml
<!--- Login -->
<string name="login_email_error">It is not an email</string>
<string name="login_email_missing">Mail is missing</string>
<string name="login_password_missing">Password is missing</string>
<string name="login_fail_login">Invalid email or password</string>
<string name="login_terms">By joining you agree to our Terms and Conditions</string>
```